Data/_render_collections.qmd

```diff
@@ -4,6 +4,9 @@ import json
 from IPython.display import display, Markdown, Latex,HTML
 from tabulate import tabulate
 
+import sys
+import os
+sys.path.append('..')
```

Comment:
This `sys.path.append('..')` looks strange to me: you want to be able to import custom files outside of the repository?

Reply:
yes, I tried some other ways, too, but since it was being rendered and function files were in different locations, it caused an issue. Maybe rearranging in order for these function files might also be needed and should be taken into consideration.

Reply:
`sys.path.append('..')` is a brittle solution as it assumes that you are working from a certain directory. I guess there must be some quarto config to just specify addtional paths to search for dependencies you want to import, but I don't know quarto well enough to figure that out at the moment.

Reply:
Sure, I can look into this aspect; there might be a way within Quarto.